client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -107,6 +125,12 @@ const DataFilesToolbar = ({ scheme }) => {
           onClick={download}
           disabled={!canDownload}
         />
+        <ToolbarButton
+          text="Link"
+          iconName="link"
+          onClick={toggleLinkModal}
+          disabled={!canDownload}
+        />
```

Comment:
Need to confirm with Tracy, but this may need to be a configurable option. If so, we can work with the solution from PR https://github.com/TACC/Frontera-Portal/pull/279

Reply:
Will wait for merge of 279 before updating